tests/admin/test_controller.py

```diff
@@ -3028,7 +3028,7 @@ def do_request(self, url, *args, **kwargs):
         """Mock HTTP get/post method to replace HTTP.get_with_timeout or post_with_timeout."""
         self.requests.append((url, args, kwargs))
         response = self.responses.pop()
-        return HTTP.process_debuggable_response(response)
+        return HTTP.process_debuggable_response(None, response)
```

Comment:
They probably changed because of changes we made as part of https://github.com/NYPL-Simplified/server_core/pull/939. It probably doesn't matter for purposes of the test, but the first argument there should become `url`.

https://github.com/NYPL-Simplified/server_core/pull/941 is a follow-up branch to fix some bugs in our 939 code; maybe review that and make sure nothing else needs to change here?

Reply:
It looks good. I'll update this to `url` for consistency. Thanks.